Automation/00_hydra/Colombia.R

```diff
@@ -289,7 +289,7 @@ rubric <- get_input_rubric()
 
 for(i in 1:10){
   # i <- 10
-  if (i != 10){ 
+  if (i < 10){ 
```

Comment:
maybe you want to add `break` in there in case of a success? Otherwise it will just keep redoing the same download, right?